alpha/scripts/utils/migrationWebcastUpdateMakeHiddenAndDisplayInSearch.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+require_once(__DIR__ . '/../bootstrap.php');
+
+if (count($argv) < 2)
+{
+    die('pleas provide partner id as input' . PHP_EOL .
+        'to run script: ' . basename(__FILE__) . ' X' . PHP_EOL .
+        'whereas X is the path to the partner ids file' . PHP_EOL);
+}
+
+$dryRun = !(count($argv) == 3 && @$argv[2] == "realrun");
+if ($dryRun)
+{
+    KalturaLog::debug('this is a dry run. pass "realrun" as 2nd param to make it actually do stuff');
+}
+
+$partners = file(@$argv[1]);
+foreach ($partners as $partnerId)
+{
+    $partnerId = trim($partnerId);
+    KalturaLog::debug("running script for partner [" . $partnerId . "]");
+    $partner = PartnerPeer::retrieveByPK($partnerId);
+    if(!$partner)
+    {
+        die('no such partner.'.PHP_EOL);
+    }
+
+    $liveEntries = array();
+
+    $c = new Criteria();
+    $c->add(entryPeer::PARTNER_ID, $partnerId);
+    $c->add(entryPeer::SOURCE, EntrySourceType::LIVE_STREAM);
+
+    $con = myDbHelper::getConnection(myDbHelper::DB_HELPER_CONN_PROPEL2);
```

Comment:
Remove this line and don't send con to the select.
Applies to second select as well.